Automattic/vip-go-mu-plugins
# Removes wp_parsely_managed_options filter from Parse.ly integration

## Description

Removes `wp_parsely_managed_options` filter from Parse.ly integration

## Changelog Description

Removes `wp_parsely_managed_options` filter from Parse.ly integration

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist

- [N/A] VIP staff: Ensure any alerts added/updated conform to internal standards (see internal documentation). 

## Steps to Test

1. Check out PR
1. Clone wp-parsely (`git clone git@github.com:Parsely/wp-parsely.git wp-parsely-3.11`)
1. Add VIP config of Parse.ly integration with only status attribute as mentioned in this PR (https://github.com/Automattic/vip-go-mu-plugins/pull/4660) 
1. Verify `wp-parsely` is enabled if status is `enabled` and UI is not any showing upgrade links
